### DISCUSSION TITLE
Experimental status of scaling modifiers

### DISCUSSION THREAD

**Discussion by RRethy** [kedacore/keda#5353] [General] [answered]
I see that .spec.advanced.scalingModifiers is still experimental, any plans on making it non-experimental? I'd like to use this feature in production, but as long as its experimental it's not something I can commit to.

**Comment by zroubalik** [comment, marked as answer]
More feedback we get from the actual usage, the sooner we will make this non-experimental. Our goal si to make it in a few months
